tide/wall/WallWindow.h

```diff
@@ -85,6 +85,9 @@ class WallWindow : public QQuickWindow
     /** Set new touchpoint's markers. */
     void setMarkers(MarkersPtr markers);
 
+    /** Set new inactivity timer. */
```

Comment:
Oh..! Actually, wouldn't InactivityTimer be a better class name than PowerTimer? Even if we don't have a PlanarController we could do some other actions with it, such as displaying a screen-saver and stoping movie playback etc...